docs/src/charm/explanation/security.md

```diff
@@ -1,2 +1,4 @@
-```{include} /snap/explanation/security.md
+# Security
```

Comment:
This creates a double header and double entry on side navigation
![image](https://github.com/user-attachments/assets/3dcd82dd-d76e-4288-9054-6d6a4508ad43)